loopy/transform/privatize.py

```diff
@@ -154,21 +154,21 @@ def privatize_temporaries_with_inames(
 
     # {{{ find ilp iname lengths
 
-    from loopy.isl_helpers import static_max_of_pw_aff
+    from loopy.isl_helpers import static_max_of_pw_aff, static_min_of_pw_aff
     from loopy.symbolic import pw_aff_to_expr
 
     priv_axis_iname_to_length = {}
     for priv_axis_inames in six.itervalues(var_to_new_priv_axis_iname):
         for iname in priv_axis_inames:
             if iname in priv_axis_iname_to_length:
                 continue
-
             bounds = kernel.get_iname_bounds(iname, constants_only=False)
             priv_axis_iname_to_length[iname] = pw_aff_to_expr(
                         static_max_of_pw_aff(bounds.size, constants_only=False))
-
-            assert static_max_of_pw_aff(
-                    bounds.lower_bound_pw_aff, constants_only=True).plain_is_zero()
+            assert (static_max_of_pw_aff(
```

Comment:
I think this was a bug that we discovered in the process of getting the PR ready